src/app/issuer/components/badgeclass-issue-bulk-award-import/badgeclass-issue-bulk-award-import.component.ts

```diff
@@ -79,11 +79,12 @@ export class BadgeClassIssueBulkAwardImportComponent extends BaseAuthenticatedRo
 				const tempColumnHeaderName: string = columnHeaderName.toLowerCase();
 				let destinationColumn: DestSelectOptions;
 
-				if (tempColumnHeaderName === 'email') {
+				if (tempColumnHeaderName === 'email' || tempColumnHeaderName === 'e-mail-adresse') {
 					inferredColumnHeaders.add('email');
 					destinationColumn = 'email';
 				}
-				if (tempColumnHeaderName === 'name') {
+
+				if (tempColumnHeaderName === 'name' || tempColumnHeaderName === 'vor/-nachname') {
```

Comment:
Isn't the case a problem here? In the CSV it's "Vor/-Nachname"

Reply:
In line 79 it is transformed to lowerCase, it works with the "N" from Nachname as well if that was your concern?